static/app/views/automations/components/automationFormData.tsx

```diff
@@ -15,30 +16,45 @@ export function getNewAutomationData(
   data: AutomationFormData,
   state: AutomationBuilderState
 ): NewAutomation {
-  const stripDataConditionIds = (condition: any) => {
+  const stripDataConditionId = (condition: any) => {
```

Comment:
nit: these functions can be extracted outside of the component so they aren't recreated on each render (not really a big deal but nice to do when you can)